CI/travis.linux.after_success.sh

```diff
@@ -4,6 +4,28 @@ set -e
 
 # we deploy only qmake and gcc combination for linux
 if [ "${Q_OR_C_MAKE}" = "qmake" ] && [ "${CC}" = "gcc" ]; then
+
+  if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+    # instead of deployment, we upload to coverity for cron jobs
+    cd build
+	  tar czf Mudlet.tgz cov-int
+    ls -l Mudlet.tgz
+    # we make this FAIL to not thrash our allowance if things go wrong!
+    curl --form token="${COVERITY_SCAN_TOKEN}" \
+      --form email=coverity@mudlet.org \
+      --form file=@Mudlet.tgz \
+      --form version="master branch head" \
+      --form description="$(git log -1|head -1)" \
+      https://scan.coverity.com/builds?project=Mudlet%2FMudlet
+    CURL_RESULT=$?
+    echo curl returned $CURL_RESULT
+    if [ $CURL_RESULT -ne 0 ]; then
+	    echo Upload to Coverity failed, curl returned $CURL_RESULT
```

Comment:
Is there \<tab>s in the file - I'm not sure those are so good in shell scripts {though mandatory in `Makefile`s}.